index.html

```diff
@@ -1,43 +1,264 @@
+      </div>
+      <p class="content-2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="btns">
+        <li>
+          <p>html</p>
+        </li>
+        <li>
+          <p>css</p>
+        </li>
+        <li>
+          <p>javascript</p>
+        </li>
+      </ul>
+      <a href="#" class="view"><p>See Project</p></a>
+    </section>
+
+    <!-- About Myself -->
+
+    <section class="wrapper-about">
+      <h2 class="about">
+        About <br />
+        Myself
+      </h2>
+      <div class="content-about">
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+      </div>
+      <a href="#" class="connect-about"><p>LET'S CONNECT</p></a>
+      <ul class="icons-about">
+        <li>
+          <a href=""><i class="fab fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-github"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-linkedin-in"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-medium-m"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="far fa-hand-peace"></i></a>
+        </li>
+      </ul>
+      <a href="#" class="view-about"><p>Get my resume</p></a>
+
+      <!-- Langguages -->
+
+      <ul class="languages-about">
+        <li>
+          <a href="#" class="language"><p>Langguages</p></a>
+        </li>
+        <li>
+          <a href="#"><i class="fas fa-chevron-down"></i></a>
+        </li>
+      </ul>
+
+      <ul class="jhc">
+        <li>
+          <a href="#" class="snapshoot-5">
+            <img src="image/js.svg" alt="js"/>
+            <p>JavaScript</p>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="snapshoot-5">
+            <img src="image/html.svg" alt="html"/>
+            <p>HTML</p>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="snapshoot-5">
+            <img src="image/css.svg" alt="css"/>
+            <p>CSS</p>
+          </a>
+        </li>
+      </ul>
+      <ul class="frameworks-about">
+        <li>
+          <a href="#" class="frameworks"><p>Frameworks</p></a>
+        </li>
+        <li>
+          <a href="#"><i class="fas fa-chevron-right"></i></a>
+        </li>
+      </ul>
+      <div class="line"><img src="image/line.svg" alt="line" /></div>
+      <ul class="frameworks-about">
+        <li>
+          <a href="#" class="frameworks"><p>Skills</p></a>
+        </li>
+        <li>
+          <a href="#"><i class="fas fa-chevron-right"></i></a>
+        </li>
+      </ul>
+      <div class="line"><img src="image/line.svg" alt="line" /></div>
+    </section>
+  </body>
 </html>
```

Comment:
- [ ] Kindly make sure your design matches the original design as much as possible, I can see some extra spaces in your page which is not giving the required UI(User Interface)😉
Your Implementation | Figma Design

![](https://user-images.githubusercontent.com/44624138/149573932-15f67150-fd5e-4df0-82cc-1a2b07c9f968.png) | ![](https://user-images.githubusercontent.com/44624138/149574027-31b4e662-741f-472f-bd34-f5aa62ab874e.png)